zip-proportional-output-fee-mechanism-pofm.rst

```diff
@@ -0,0 +1,244 @@
+
+The change to the conventional transaction fees must be undertaken soon
+as the Zcash network has been under heavy load with high-output transactions while 
+regular shielded transactions with 2 outputs are affected when relying on current 
+light wallet infrastructure.
+
+The following parties need to be part of the consensus:
+
+* The technical aspects of a changing conventional fee based on outputs 
+need to be evaluated.
+* A guarantee from mining groups is required to include the updated POFM 
+transactions in the next block.
+* Wallet developers need to update the software to use the new fee.
+* Zcash documentation and community outreach must be undertaken to
+make the change known.
+
+
+Requirements for adoption
+-------------------------
+
+The change to the conventional transaction fees should be undertaken soon
+as it gets difficult to gain consensus with the growth in the network
+of wallets, exchanges, miners, and third parties involved.
+
+The following parties need to be part of the consensus:
+
+* Support from mining groups is required to include the updated conventional
+  fee transactions in the next block.
+* Wallet developers need to provide a commitment to update the software to use
+  the new fee.
+* Zcash documentation and community outreach must be undertaken to make the
+  change known.
+* Changes to librustzcash crates to:
+  * expose minimum recommended fee for a given number of recipients
+  * implement default fee equation and corresponding unit tests
+  * integration of such mechanism to Transaction Builder APIs
+* adopt `librustzcash` changes to mobile SDK repos:
+  * ZcashLightClientKit
+  * zcash-light-client-ffi
+  * zcash-android-wallet-sdk
+
+
+Specification
+=============
+
+Wallets implementing this specification will use a conventional fee in the form of 
+base_fee and marginal_fee.
+
+The proposal would end up selecting one of:
+* base_fee = 1000, marginal_fee = 250 in @nuttycom's proposal.
+* base_fee = 1000, marginal_fee = 1000 in @madars' proposal.
+* base_fee = 10000, marginal_fee = 2500 in @daira's proposal.
+* base_fee = 1000, marginal_fee = 1000 for Shielded, Shielding and De-shielding
+transactions, and base_fee = 10000, marginal_fee = 10000 for Transparent transactions 
+per @nighthawk24's proposal.
+
+And calculated per the following forumla:
+min_fee = base_fee + marginal_fee * max(0, inputs + outputs - grace_window_size)
+
+where grace_window_size = 4
+
+
+Security and Privacy considerations
+=============
+
+Unique transaction fees may reveal specific users or wallets or wallet versions,
+which would reduce privacy for those specific users and the rest of the network.
+Hence this change should be accepted by a majority of shielded transaction
+software providers before deploying the change.
+
+Long term, the issue of fees needs to be re-visited in separate future
+proposals as the blocks start getting consistently full. New ZIPs with 
+scaling solutions, will need to be evaluated and applied.
```

Comment:
This isn't really a Security or Privacy Consideration, but instead either "Unresolved Questions" or "Future Possibilities" (categories from the Rust RFC process that seem applicable here). Consider either moving this to one of those sections, or folding it into the Motivation section (something like "This is intended as a short-to-medium-term solution. Long term...").